--- a/Source/CTest/cmCTestBuildHandler.cxx
+++ b/Source/CTest/cmCTestBuildHandler.cxx
@@ -570,7 +570,7 @@ public:
   FragmentCompare(cmFileTimeComparison* ftc): FTC(ftc) {}
   FragmentCompare(): FTC(0) {}
-  bool operator()(std::string const& l, std::string const& r)
+  bool operator()(std::string const& l, std::string const& r) const
     {
     // Order files by modification time.  Use lexicographic order
     // among files with the same time.
